src/app/modules/consultations/consultations-summary/consultations-summary.component.ts

```diff
@@ -99,8 +99,8 @@ export class ConsultationsSummaryComponent implements OnInit {
 
   splitResponses(responsesList) {
     if (responsesList.length) {
-      this.annonymousResponses = responsesList.filter(response => (response.node.user === null || response.node.user === undefined));
-      this.publicResponses = responsesList.filter(response => response.node.user !== null);
+      this.annonymousResponses = responsesList.filter(response => !response.node.user);
+      this.publicResponses = responsesList.filter(response => response.node.user);
```

Comment:
1. The `filter` can be removed as there is no filtration going on anymore
2.  Not sure if `annonymousResponses` has any use anymore as all responses are essentially public

Reply:
1. We are filtering based on if a user object is present or not. If a user object is present, the response will be a public response if not the response will be an anonymous response. Here I haven't changed the logic, just refactored the code.

2. The treatment and use of anonymous response remains the same, we have just removed the `Unverified` tag from the response.

Reply:
Ah got it, missed the `!`

It doesn't look like it's related to the tag removal though. What was the reason behind this change? The old code seems to be doing the same job

Reply:
Discussed on call: This is just an improvement we made